Dockerfile

```diff
@@ -0,0 +1,4 @@
+# Docker image for running gdal/ogr and fetch (wget) commands for ogdc recipes.
+FROM ghcr.io/osgeo/gdal:alpine-normal-latest
+# We use wget to fetch data from remote sources.
+RUN apk update && apk add wget
```

Comment:
The only reason we need a custom docker image for the simple examples I've been testing against is the need to fetch data with `wget`. Most docker images don't come with it, and some (like `alpine`, as in this case) ship with the busybox version. This lacks some options we need for QGreenland data layers (`--content-disposition`), including the [seal-tags](https://github.com/QGreenland-Net/ogdc-recipes/blob/e4c5e3aebc83ccc68da9170106dce9ecc9634c54/recipes/seal-tags/meta.yml#L9) example.

Perhaps there's a different/better way to approach fetching HTTP assets. I considered using `hera`'s `HTTPArtifact` as input, but this also lacks the option of fetching a remote asset into a specified directory without giving the fetched thing a name. 

Longer term, I imagine we'll want other options as well (and maybe even want to use `hera`'s [RunnerScriptConstructor](https://hera.readthedocs.io/en/stable/api/workflows/hera/?h=runnerscript#hera.workflows.RunnerScriptConstructor) (see also: https://hera.readthedocs.io/en/stable/user-guides/script-basics/) ). So this image may very well grow more complex.